# This message describ the grasp server request

# The global PointCloud
sensor_msgs/PointCloud2 global_cloud

# Coordinates of the object bounding box
grasping_tp/BoundingBoxCoord bounding_box